posix.mak

```diff
@@ -1,25 +1,129 @@
-# Makefile to build the entire dlang.org website
+# dlang.org Makefile
+# ==================
 #
-# To run:
+#  This Makefile is used to build the dlang.org website.
+#  To build the entire dlang.org website run:
 #
-# make -f posix.mak all
+#  	make -f posix.mak all
 #
-# To also upload to the website:
+#  Build flavors
+#  -------------
 #
-# make -f posix.mak rsync
+#  This makefile supports 3 flavors of documentation:
 #
+#  	latest			Latest released version (by git tag)
+#  	prerelease		Master (uses the D repositories as they exist locally)
+#  	release			Documentation build that is shipped with the binary release
 #
-# This makefile supports 3 flavors of documentation, latest, prerelease, and
-# release. The first 2 are live on the website and built from the latest
-# released version and master (prerelease). The release build is used for the
-# documentation that is shipped with binary releases. For the latter the LATEST
-# version is not yet published at build time, hence a few things differ from a
-# prerelease build.
+#  For `release` the LATEST version is not yet published at build time,
+#  hence a few things differ from a `prerelease` build.
 #
-# To build latest and prerelease docs:
-#   make -f posix.mak all
-# To build release docs:
-#   make -f posix.mak RELEASE=1 release
+#  To build `latest` and `prerelease` docs:
+#
+#    make -f posix.mak all
+#
+#  To build `release` docs:
+#
+#    make -f posix.mak RELEASE=1 release
+#
+#  Individual documentation targets
+#  --------------------------------
+#
+#  The entire documentation can be built with:
+#
+#    make -f posix.mak docs
+#
+#  This target is an alias for two targets:
+#
+#  A) `docs-prerelease` (aka master)
+#
+#    The respective local repositories are used.
+#   	This is very useful for testing local changes.
+#   	Individual targets include:
+#
+#    	dmd-prerelease
+#    	druntime-prerelease
+#    	phobos-prerelease
+#    	apidocs-prerelease		Ddox documentation
+#    	apidocs-serve			Starts a development Ddox webserver
+#    							(It will (re)-generate requested documentation pages on-the-fly)
+#
+#  B) `docs-latest` (aka stable)
+#
+#  Based on the last official release (git tag), the repositories are freshly cloned from GitHub.
+#  Individual targets include:
+#
+#   	dmd-latest
+#   	druntime-latest
+#   	phobos-latest
+#   	apidocs-latest			Ddox documentation
+#
+#  Options
+#  -------
+#
+#  Most commonly used options include:
+#
+#   	DIFFABLE=1			Removes inclusion of all dynamic content and timestamps
+#   	RELEASE=1			Release build (needs to be set for the `release` target)
+#   	LATEST=2.072.1		Specify the LATEST version manually. Avoids network traffic.
```

Comment:
LATEST no longer (thankfully) requires network traffic to get. It's specified in the VERSION file. I'm not sure if there is any reason to override it on the command line any more.